javascript/index.js

```diff
@@ -32,34 +40,66 @@ function printMilliseconds() {
 
 function printSplit() {
   // ... your code goes here
+
+  let listItem = document.createElement('li');
+  splitsElement.appendChild(listItem);
+  listItem.innerHTML = chronometer.split();
 }
 
 function clearSplits() {
   // ... your code goes here
+  splitsElement.innerHTML = ' ';
```

Comment:
Here it should be an empty string (""), " " has a space which is not an empty string